### PR TITLE
chore: update packages version

### PR DESCRIPTION
**Pull request by Unix4ever**
Via renovate.

Signed-off-by: Artem Chernyshev <artem.chernyshev@talos-systems.com>

### CODE REVIEW

File: Pkgfile [hunk 1]
```diff
@@ -164,9 +164,9 @@ vars:
   arm_trusted_firmware_sha512: 3e1e09bcc69a0f080f91b94d9c8da6dd0b27b86292599c280438d7b6f594017a896af7cfc58d9d22053f77b5daa2faecd51a59505c755eac8493e4e0fd1e9203
 
   # renovate: datasource=github-tags extractVersion=^v(?<version>.*)$ depName=u-boot/u-boot
-  uboot_version: 2022.07
```
Comment (smira) [inline Pkgfile:167]: please don't bump u-boot, I think Noel said we need to update patches which we don't have yet

Comment (Unix4ever) [reply Pkgfile:167]: ah, ok

Comment (Unix4ever) [reply Pkgfile]: reverted that

**Review by smira — APPROVED** [review]

**Pull request merged**